doc/source/user_guide/tutorial_parallelization_dask.ipynb

```diff
@@ -0,0 +1,364 @@
+    "\n",
+    "If you need to access a particular image you can do so by slicing the list with `images[index]`. \n",
+    "Calling `compute` will cause the image to be loaded and stored into memory.\n",
+    "\n",
+    "```python\n",
+    "image_of_interest = images[index].compute()\n",
+    "```\n",
+    "\n",
+    "Note that on calls to `compute`, Dask is re-computing all of the computation (in this case, just loading). \n",
+    "This is likely acceptable since computing individual images might be rather quick compared.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "input_variances = []\n",
+    "output_variances = []\n",
+    "output_images = []\n",
+    "\n",
+    "for image in images:\n",
+    "    input_variance = delayed(np.var)(image)\n",
+    "    # Now we compute the output image as a processed version off the input\n",
+    "    # image\n",
+    "    output_image = delayed(filters.gaussian)(image, 10)\n",
+    "    output_variance = delayed(np.var)(output_image)\n",
+    "    output_images.append(output_image)\n",
+    "\n",
+    "    # Store all the results into the arrays we care about\n",
+    "    input_variances.append(input_variance)\n",
+    "    output_variances.append(output_variance)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "saved_list = []\n",
+    "for i, image in enumerate(output_images):\n",
+    "    image_ubyte = delayed(img_as_ubyte)(image)\n",
+    "    filename = os.path.join(save_directory,\n",
+    "                            'image_{i:2d}.bmp'.format(i=i))\n",
+    "    saved_list.append(delayed(imageio.imwrite)(filename, image_ubyte))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now visualize what the computation looks like. Visualizing the whole batch might be a little daunting so we will instead visualize the first 3 elements of our computation. You will need to install `python-graphviz` for these next few lines to execute."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dask\n",
+    "# uncomment the line below to visualize the graph\n",
+    "# dask.visualize(delayed([input_variances[:3], output_variances[:3], saved_list[:3]]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Visualizing the computation graph is a good way to double check your code. In fact, it helped me find a bug in mine! Here, the important aspect of the graph is that the data paths for the analysis is completely independent from one image to the next. This is what will help us get the most from parallelization.\n",
```

Comment:
I would remove the sentence "In fact...". Too informal, and does not provide any useful information.